### DISCUSSION TITLE
Some Nonsencial Thoughts: Adding GPT Techniques will be Interesting

### DISCUSSION THREAD

**Discussion by clean99** [autometrics-dev/autometrics-shared#72] [Ideas]
Data + GPT would be a good combo. Adding GPT in the product more or less can help improve functionality. For example: using GPT to analyze functions' latency, error rate, and code, to provide user suggestions about how to improve the code.

**Comment by brettimus** [comment]
this is something i'm also very interested in :)

combining observability data with code that produced said data seems very promising. i'm curious what you think would be a good ux for this? i was imagining something close to the way linters add somewhat unobtrusive warnings/hints, but there's a lot of different possibilities
